spark/build.gradle.kts

```diff
@@ -53,6 +60,18 @@ android {
     kotlinOptions {
         jvmTarget = "11"
         allWarningsAsErrors = true
+        freeCompilerArgs = freeCompilerArgs + "-opt-in=com.adevinta.spark.InternalSparkApi"
+        freeCompilerArgs = freeCompilerArgs + "-opt-in=com.adevinta.spark.ExperimentalSparkApi"
+        freeCompilerArgs = freeCompilerArgs + "-opt-in=kotlin.RequiresOptIn"
+    }
+
+    sourceSets.configureEach {
+        kotlin.srcDir("build/generated/ksp/${name}/kotlin")
```

Comment:
Is this the way to do it?

Reply:
It depends. Do we really need to access the generated code from our own code?
If no, I suggest we remove this entirely.